### PR TITLE
fix: wait for Target.fetchFromTarget command

### PR DESCRIPTION
**Pull request by MorbidAngel**
This change solves the problem with:

```
asyncio default_exception_handler :1604 Future exception was never retrieved
future: <Future finished exception=NetworkError('Protocol error (Target.detachFromTarget): No session with given id')>
```

error.

### CODE REVIEW

**Comment by Mattwmaster58** [comment]
Hi @MorbidAngel, thanks for the PR! Unfortunately, it seems this has already been implemented in #16.

**Comment by MorbidAngel** [comment]
@Mattwmaster58 Hi, sorry I've missed that. Is it going to be merged into dev branch soon ?. Or should I use the pup2.1.1. branch instead (is this branch `pup2.1.1` stable enough to be used in the production) ?

**Comment by MorbidAngel** [comment]
> Hi @MorbidAngel, thanks for the PR! Unfortunately, it seems this has already been implemented in #16.

Sorry, but I don't see any related change in the https://github.com/pyppeteer/pyppeteer/pull/16/files#diff-5c728b692de8dbba0a50ecc6e6df6398R124.

**Comment by Mattwmaster58** [comment]
Sorry about the false alart, I have to admit it's a knee jerk reaction to seeing a PR that isn't merging to pup2.1.1 just because of all the work we've done. If you could set the destination to pup2.1.1 and make that appropriate changes that would be great.

**Comment by MorbidAngel** [comment]
Not a problem,  I've changed the base branch for this PR to the `pup2.1.1`. But still, I repeat my question - is `pup2.1.1` stable enough to be used on the production ?

**Comment by Mattwmaster58** [comment]
No, I wouldn't use it in production yet, even though I would consider it fairly stable for simple tasks.

File: pyppeteer/page.py [hunk 1]
```diff
@@ -116,12 +116,12 @@ def __init__(
...
-        def _onTargetAttached(event: Dict) -> None:
+        async def _onTargetAttached(event: Dict) -> None:
             targetInfo = event['targetInfo']
             if targetInfo['type'] != 'worker':
                 # If we don't detach from service workers, they will never die.
                 try:
-                    client.send('Target.detachFromTarget', {'sessionId': event['sessionId'],})
+                    await client.send('Target.detachFromTarget', {'sessionId': event['sessionId'],})
                 except Exception as e:
                     logger.error(f'An exception occurred: {e}')
```
Comment (Mattwmaster58) [inline pyppeteer/page.py]:
```suggestion
                except Exception as e:
                    logger.exception(f'An exception occurred while trying to detach from target')
```

**Review by Mattwmaster58 — CHANGES_REQUESTED** [review]
Nothing wrong with your implementation, I just couldn't resist improving the logging here 😉

**Review by Mattwmaster58 — APPROVED** [review]

**Pull request merged**